frontend/src/API/callDefinitions.ts

```diff
@@ -1,9 +1,12 @@
+import store from '../store';
+
 type fetchTypes = 'POST' | 'PUT' | 'DELETE';
 
 const genericFetch = (fetchType: fetchTypes, url: string, data: any) => {
   return fetch(url, {
     body: JSON.stringify(data),
     headers: {
+      Authorization: `Bearer ${store.getState().oidc.user.id_token}`,
```

Comment:
With testing my suspicions: I suspect it has to be as it is to call getState the moment it actually fetches instead of before trying to fetch. In case oidc.user.id_token has changed in the mean time.